service-api/module/Application/src/Services/Experian/FraudApi/DTO/ExperianCrosscoreFraudResponseDTO.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Services\Experian\FraudApi\DTO;
+
+use Application\Services\Experian\FraudApi\ExperianCrosscoreFraudApiException;
+use phpDocumentor\Reflection\PseudoTypes\Numeric_;
+
+class ExperianCrosscoreFraudResponseDTO
```

Comment:
This DTO is returning pretty much everything, but we don't need all this information. It would be good to cut it down to just the bits we actually want, which would also make the tests a lot neater because there wouldn't be as much unused info in there.